gadgets/fsnotify/program.bpf.c

```diff
@@ -40,6 +40,49 @@ struct process {
 	char pcomm[TASK_COMM_LEN];
 };
 
+enum i_mask_set : u32 {
+ 	IN_ACCESS =	0x00000001,
+ 	IN_MODIFY =	0x00000002,
+ 	IN_ATTRIB =	0x00000004,	
+ 	IN_CLOSE_WRITE = 0x00000008,
+	IN_CLOSE_NOWRITE = 0x00000010,
+	IN_OPEN	= 0x00000020,
+	IN_MOVED_FROM =	0x00000040,
+	IN_MOVED_TO	= 0x00000080,
+	IN_CREATE = 0x00000100,
+	IN_DELETE =	0x00000200,
+	IN_DELETE_SELF = 0x00000400,
+	IN_MOVE_SELF = 0x00000800,
+	IN_UNMOUNT = 0x00002000,
+	IN_Q_OVERFLOW =	0x00004000,
+	IN_IGNORED = 0x00008000,
```

Comment:
Shouldn't this list be continued until IN_ONESHOT=0x80000000?
https://github.com/torvalds/linux/blob/master/include/uapi/linux/inotify.h#L30

For example, an inotify event could have `IN_ISDIR=0x40000000` according to [man inotify.7](https://man7.org/linux/man-pages/man7/inotify.7.html).